math/orpie/Portfile

```diff
@@ -18,17 +18,39 @@ long_description    Orpie is a fullscreen RPN calculator for the console. Its \
                     functionality, command completion, and a visible interactive \
                     stack.
 
-homepage            http://pessimization.com/software/orpie/
-master_sites        ${homepage}
+# not actually used for compiling -- OPAM is used instead
+homepage            https://github.com/pelzlpj/orpie/
+master_sites        ${homepage}/archive/refs/tags/
+distname            release-${version}
+worksrcdir          orpie-release-${version}
```

Comment:
Use the github portgroup and its `github.setup` procedure to properly set up projects hosted at GitHub. Setting these variables manually is then probably not necessary.

Reply:
The project is hosted at GitHub, but it is not actually fetched by port, but instead by [opam](https://opam.ocaml.org/).